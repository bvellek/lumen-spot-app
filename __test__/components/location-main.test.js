import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LocationMain from '../../public/js/components/location-main';

describe('<LocationMain />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <LocationMain />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
