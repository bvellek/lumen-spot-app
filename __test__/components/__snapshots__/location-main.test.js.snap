// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationMain /> it renders correctly 1`] = `
<main>
  <LocationData />
  <Connect(ImageResultsContainer) />
</main>
`;
